atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/asts/DynamicTaskScheduler.java

```diff
@@ -41,7 +42,7 @@ private DynamicTaskScheduler(
             ScheduledExecutorService scheduledExecutorService,
             Refreshable<Duration> automaticSweepRefreshDelay,
             Runnable task,
-            @CompileTimeConstant String safeLoggableTaskName) {
+            String safeLoggableTaskName) {
```

Comment:
Removed compile time constant requirement because it is fine to derive these names (and we will need to do that - e.g. sweeper-1)